ggml/src/ggml-sycl/ggml-sycl.cpp

```diff
@@ -54,18 +54,12 @@ static ggml_sycl_device_info ggml_sycl_init() {
     GGML_ASSERT(info.device_count <= GGML_SYCL_MAX_DEVICES);
 
     int64_t total_vram = 0;
-#if defined(GGML_SYCL_FORCE_MMQ)
-    GGML_LOG_INFO("%s: GGML_SYCL_FORCE_MMQ:   yes\n", __func__);
-#else
-    GGML_LOG_INFO("%s: GGML_SYCL_FORCE_MMQ:   no\n", __func__);
-#endif
-#if defined(SYCL_USE_XMX)
-    GGML_LOG_INFO("%s: SYCL_USE_XMX: yes\n", __func__);
-#else
-    GGML_LOG_INFO("%s: SYCL_USE_XMX: no\n", __func__);
-#endif
-    GGML_LOG_INFO("%s: found %d %s devices:\n", __func__, info.device_count, GGML_SYCL_NAME);
-
+/* This is a bit misleading;  reserved for later */
+// #if defined(SYCL_USE_XMX)
+//     GGML_LOG_INFO("%s: SYCL_USE_XMX: yes\n", __func__);
+// #else
+//     GGML_LOG_INFO("%s: SYCL_USE_XMX: no\n", __func__);
+// #endif
```

Comment:
As far as I can tell the project never queries for `sycl::aspect::ext_intel_matrix` but relies on the macro `SYCL_USE_XMX` to change some tiles parameters. As this [TODO](https://github.com/ggerganov/llama.cpp/blob/master/ggml/src/ggml-sycl/common.hpp#L68) says it is not a great name for it but I think it makes sense to log whether `SYCL_USE_XMX` is set or not.
I think printing whether the device has `sycl::aspect::ext_intel_matrix` is more misleading.

Reply:
`SYCL_USE_XMX` is only set when `GGML_SYCL_FORCE_MMQ` is not defined:

https://github.com/ggerganov/llama.cpp/blob/f8feb4b01af374ad2fce302fd5790529c615710b/ggml/src/ggml-sycl/common.hpp#L67-L71

The comment mentions that it is not being used for SYCL currently. It also seems like it is always true since mmq kernels are disabled/faulty.

`sycl::aspect::ext_intel_matrix` checks whether hardware supports XMX or not. I am not sure how that is misleading. We can re-enable these `SYCL_USE_XMX` after proper implementation.

Reply:
Thanks for the explanations. Maybe it's not that useful to log for `SYCL_USE_XMX` as long as it is tied to `GGML_SYCL_FORCE_MMQ` which is already logged.

I wasn't sure what was the point for querying `sycl::aspect::ext_intel_matrix` but if some dependencies rely on it why not.

Reply:
Currently, `sycl::aspect::ext_intel_matrix` is only used for displaying if a GPU has XMX or not in `ggml_backend_sycl_print_sycl_devices` function replacing `SYCL_USE_XMX: yes/No` logs.

I want to expand this to other GPUs too but I am having hard time finding proper documentation.
In the future, I was thinking to cache it and make it useful for writing kernels with joint_matrix extensions for such devices.

Let's see what Intel team has to say. This is not important for now and can be removed.